src/lambda/wizards/samDeployWizard.ts

```diff
@@ -512,6 +516,10 @@ export class SamDeployWizard extends MultiStepWizard<SamDeployWizardResponse> {
 
         return this.response.stackName ? undefined : this.S3_BUCKET
     }
+
+    private skipOrReturnRegion(skipToStep: WizardStep | undefined): WizardStep | undefined {
```

Comment:
"return region" is misleading to me, would `skipOrPromptRegion` make sense?

Reply:
TECHNICALLY everything here is returning steps but that probably makes more sense to the observer.

Reply:
up to you